func/ui.c

```diff
@@ -118,76 +111,25 @@ int requestRecipeNumber(){
     return recipeNumber -1;
 }
 
-/**
- * Prints information about a given recipe.
- */
-void printRecipe(const Recipe *recipe, const int people){
     /**
      * For mysterious reasons, the last digit of "cellLengths" becomes really big, and wrecks the system.
      * This issue is solved with a hack: add an extra item at the end of the array, and don't read it.
      */
-    int cellLengths[] = {22, 14, 14, 0};
-    const int numberOfCells = sizeof(cellLengths) / sizeof(cellLengths[0]) - 1;
+void printRecipe(Recipe *recipe, const int people){
 
-    const int titleLength = 50;
-    char title[titleLength+1];
+    printf("\n\n");
+    printf("----------------------------------------------------------\n");
     char name[MAX_RECIPE_NAME];
     strcpy(name, recipe->name);
-    capitaliseFirst(name);
-    padAround(name, title, titleLength);
-
-    printf("\nHere you go:\n");
-    printLine("┌", "─", "┐", numberOfCells, cellLengths);
-    printf("│ %s │", title);
-    printLine("├", "┬", "┤", numberOfCells, cellLengths);
+    printf("|                        %-10s                      |\n", name);
+    printf("|--------------------------------------------------------|\n");
     for (int i = 0; i < recipe->ingredientCount; i++){
-        char ingredientName[MAX_INGREDIENT_NAME];
-        strcpy(ingredientName, recipe->ingredients[i].name);
-        capitaliseFirst(ingredientName);
-
-        //TODO: print the correct g CO₂ number
-        printf("│ %-20s │ %9.1lf(g) │ %5d(g CO₂) │", ingredientName, recipe->ingredients[i].amount * people , 25);
-        if(i < recipe->ingredientCount - 1) printf("\n");
+        //TODO: do the number
+        printf("| %-20s|%10f(g)     |%6d(g CO₂)  |\n", capitaliseFirst(name), recipe->ingredients[i].amount*people , 25);
```

Comment:
```suggestion
        printf("│ %-20s │ %9.1lf(g) │ %5d(g CO₂) │\n", capitaliseFirst(name), recipe->ingredients[i].amount*people , 25);
```
Hvorfor fjerner I en forbedring fra #38? (som vi blev enige om at lave i #37)